src/installer/tests/HostActivation.Tests/PortableAppActivation.cs

```diff
@@ -332,6 +332,36 @@ public void AppHost_FrameworkDependent_GlobalLocation_Succeeds(bool useRegistere
             }
         }
 
+        [Fact]
+        public void RuntimeConfig_FilePath_Breaks_MAX_PATH_Threshold()
+        {
+            var project = sharedTestState.PortableAppFixture_Published
+                .Copy();
+
+            var appExeName = Path.GetFileName(project.TestProject.AppExe);
+            var outputDir = project.TestProject.OutputDirectory;
+
+            // Move the portable app to a path such that the length of the executable's fullpath
+            // is just 1 char behind MAX_PATH (260) so that the runtimeconfig(.dev).json files
+            // break this threshold. This will cause hostfxr to normalize these paths -- here we
+            // are checking that the updated paths are used.
+            var dirName = new string('a', 259 - outputDir.Length - appExeName.Length - 2);
```

Comment:
Nit: In theory this would break if the location of the repo was in a too deep. Maybe we should create a folder in temp (which is basically guaranteed to the short enough on all systems).

We do have other tests creating long paths, but those don't need specific path length, they just need path longer than something. This test needs a specific path length and so the arithmetic above could end up with negative size (and the test will fail).

Reply:
Yeah, this totally relies on the user having the repo on a path not so far from root. Wasn't sure if it was good to put this on %TMP%, now I remember that we do that for bundling tests... changing this.